engine/api/src/main/scala/pl/touk/nussknacker/engine/defaults/deployment/DefaultProcessStateConfigurator.scala

```diff
@@ -0,0 +1,38 @@
+package pl.touk.nussknacker.engine.defaults.deployment
```

Comment:
1. please replace order of package - should be `deployment.defaults` - take a look at api.dict.embedded
2. we should use singular package names instead of plural
3. what about replacing defaults with something like "simple"

Reply:
I've moved it to simple in development.

Reply:
Moved